sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -241,32 +283,57 @@ def _fit(self, y, X, fh):
             )
 
         if self.validation_split is not None:
-            split = int(len(y) * (1 - self.validation_split))
+            if X is None:
+                y_train, y_test = temporal_train_test_split(
+                    y, test_size=self.validation_split
+                )
+            else:
+                y_train, y_test, X_train, X_test = temporal_train_test_split(
+                    y, X, test_size=self.validation_split
+                )
 
             train_dataset = PyTorchDataset(
-                y[:split],
+                y_train,
                 config.context_length + max(config.lags_sequence),
-                X=X[:split] if X is not None else None,
+                X=X_train if X is not None else None,
                 fh=config.prediction_length,
+                batch_size=(
+                    self._training_args["per_device_train_batch_size"]
+                    if "per_device_train_batch_size" in self._training_args.keys()
+                    else 8
```

Comment:
I am not a big fan of such magic number. Could you set the per_device_train_batch_size in the __init__ method if it is not a part of _training_args? 
Furthemore, please document it in the doctoring of the class that the default will be 8 in this case.